browser_extension/js/main.js

```diff
@@ -24,12 +24,20 @@ $(function () {
   var $currentInput = $usernameInput.focus();
 
   var socket = io.connect('http://localhost:3000');
+  
+  var currentchannel;
 
   // Payload stores the username and channel
   var payload = {
     username: "",
     domain: ""
   }
+  
+  chrome.tabs.query({'active': true, 'lastFocusedWindow': true}, function (tabs) {
```

Comment:
Your implementation replaces the payload.domain in `authentication.js`
You can delete the following dead code from that file:
- [browser_extension/src/bg/authentication.js#L57-L62](https://github.com/marcobeltempo/chaddon/blob/dev/browser_extension/src/bg/authentication.js#L57-L62)
- [browser_extension/src/bg/authentication.js#L71](https://github.com/marcobeltempo/chaddon/blob/dev/browser_extension/src/bg/authentication.js#L71)
- [browser_extension/src/bg/authentication.js#L77](https://github.com/marcobeltempo/chaddon/blob/dev/browser_extension/src/bg/authentication.js#L77)